.github/workflows/fix_doctests.yml

```diff
@@ -0,0 +1,27 @@
+name: fix_doctests
+on:
+  pull_request:
+jobs:
+  doctests:
+    name: Fix doctests (Julia ${{ matrix.julia-version }} - ${{ github.event_name }})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        julia-version: [1.6]
+    steps:
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: ${{ matrix.julia-version }}
+      - uses: actions/checkout@v1
+      - name: Fix doctests
+        shell: julia --project=docs/ {0}
+        run: |
+          using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()
+          using Documenter
+          using ChainRulesTestUtils
+          doctest(ChainRulesTestUtils, fix=true)
```

Comment:
Might also want to use the same trick as in https://github.com/JuliaDiff/FiniteDifferences.jl/pull/173/commits/7077850ca5e92ec8caa459174fc3e7b2004c54a1 here

Reply:
maybe, but we have to roll that out in many places so I will wait til we do that generally everywhere

Reply:
Yeah, we can wait until we run into any issues.